MetaMask/metamask-extension
# test: Complete missing step for add a contact to the address book in existing E2E test

## **Description**

Adding missing "add contact" step to this E2E test `test/e2e/tests/settings/address-book.spec.js` to fully cover the manual scenario [here](https://github.com/MetaMask/metamask-extension/blob/develop/test/manual-scenarios/address-book/add%20a%20contact%20to%20the%20address%20book.csv).

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27959?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/27369

## **Manual testing steps**

1. Run yarn test:e2e:single test/e2e/tests/settings/address-book.spec.js --browser=chrome --leave-running
2. Test should pass with no failure
3. The second test run in the test suite is the one added in this PR

_Note: I have used the class instead of specific selectors (text+tag) for the element as the latter (Add contact/button) would not be located when running the test and the test would consistently fail._ 

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
